server/services/nl.py

```diff
@@ -89,9 +89,15 @@ def detect_svs(self, query, embeddings_build):
 
   def detect_place(self, query):
     doc = self.ner_model(query)
-    places_found = []
+    places_found_loc_gpe = []
+    places_found_fac = []
     for e in doc.ents:
+      # Preference is given to LOC and GPE types.
       if e.label_ in ["GPE", "LOC"]:
```

Comment:
nit: Are these entity types? Would be helpful to add a comment for inexperienced people like me :)

Reply:
Good point. Added comments.